### ISSUE TITLE
508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#15772]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

- Operating System: all
- Browser: all
- Screenreading device: any
- Server destination: staging & production

## Steps to Recreate

1. Enter [https://www.va.gov/find-forms/] in browser
2. Have developer tools open, and the axe browser extension loaded
3. Run an axe audit
4. Verify that [Change your direct deposit information] is called out as an error of "Heading levels should only increase by one"
5. This error is repeated throughout the component

### Sample 

**Current code**
```html
<h4 class="va-nav-linkslist-title">Change your direct deposit information</h4>
```

**Recommended code**
```html
<h3 class="vads-u-font-size--h4">Change your direct deposit information</h3>
```

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)

## Screenshots

![Screen Shot 2020-11-04 at 9 49 41 AM](https://user-images.githubusercontent.com/57469/98136666-2f70a480-1e8f-11eb-8c02-8796d98e76c7.png)

![Screen Shot 2020-11-04 at 9 48 50 AM](https://user-images.githubusercontent.com/57469/98136732-3eefed80-1e8f-11eb-95c7-8ce1c3fdf6c6.png)

**Comment by zacharymorel** [comment]
Thread to current collaboration to determine best solution => https://dsva.slack.com/archives/C01DWJFD45D/p1605813270016600

**Comment by zacharymorel** [comment]
Good afternoon  @jenstrickland this is ready to be validated on https://staging.va.gov/ .

**Comment by jenstrickland** [comment]
Validated 🎉
